messaging/src/main/java/org/axonframework/queryhandling/QueryUpdateEmitter.java

```diff
@@ -96,7 +100,7 @@ private static <U> SubscriptionQueryUpdateMessage<U> asUpdateMessage(Object payl
         } else if (payload instanceof Message) {
             return new GenericSubscriptionQueryUpdateMessage<>((Message<U>) payload);
         }
-        // todo: use MessageNameResolver below
+        // TODO #3085 use MessageNameResolver below
```

Comment:
Trusting that this will be picked up in another PR, right?

Reply:
Yes